src/System.Runtime.InteropServices.RuntimeInformation/src/Resources/Strings.resx

```diff
@@ -61,4 +61,7 @@
   <data name="Argument_EmptyValue" xml:space="preserve">
     <value>Value cannot be empty.</value>
   </data>
+  <data name="PlatformNotSupported_RuntimeInformation" xml:space="preserve">
+    <value>RuntimeInformation is a stub implementation assembly for PCLs, where runtime rids are not supported by packages.config.</value>
```

Comment:
+1 this is a really bad message. Perhaps `RuntimeInformation is not supported for Portable Class Libraries.`